src/PrestaShopBundle/Install/Install.php

```diff
@@ -311,22 +311,18 @@ public function installDatabase($clear_database = false)
             return false;
         }
 
-        return $this->generateSf2ProductionEnv();
+        return $this->generateSf2Env();
     }
 
     /**
-     * Pass SF2 to production
      * cache:clear
      * assetic:dump
      * doctrine:schema:update.
      *
      * @return bool
      */
-    public function generateSf2ProductionEnv()
+    public function generateSf2Env()
```

Comment:
This is a BC break

Reply:
Right, its not necessary though, just method doesnt seem to fo anything specific to production though. Do you think i should leave original name?

Reply:
ok I've left the original name. But if you agree to BC break, I guess It should at least be more meaningful.
Smth like `updateSchema()` ?. Let me know what you think.

Reply:
I totally agree about the BC break, it was about adding it in the PR's description ^^'
` updateSchema`  seems to be a very good idea